h5py/tests/__init__.py

```diff
@@ -18,6 +18,6 @@ def run_tests(args=''):
         from shlex import split
         from subprocess import call
         from sys import executable
-        cli = [executable, "-m", "pytest", "--pyargs", "h5py"]
+        cli = [executable, "-m", "pytest", "--pyargs", "h5py", "-x"]
```

Comment:
We probably want to allow users to run the full test suite, rather than failing on the first test. Calling `run_tests("-x")` would be equivalent to this change.

Reply:
reverted with b803a52